pkg/cmd/server/api/v1/types.go

```diff
@@ -338,6 +338,11 @@ type ImagePolicyConfig struct {
 	// MaxScheduledImageImportsPerMinute is the maximum number of scheduled image streams that will be imported in the
 	// background per minute. The default value is 60. Set to -1 for unlimited.
 	MaxScheduledImageImportsPerMinute int `json:"maxScheduledImageImportsPerMinute"`
+	// AllowedRegistriesForImport is a white list of registries that we allow to import the
+	// images from. By default docker.io, Red Hat registry, gcr.io, Amazon ECS, quay.io are
+	// allowed. If you want to import images from your private Docker Registry, you have to
+	// add it into the list.
```

Comment:
This needs to only impose this behavior when this is set.  If empty, old behavior applies.